web/locales/en/messages.ftl

```diff
@@ -774,24 +774,24 @@ help-reach-hours-pluralized = Help reach { NUMBER($hours) ->
 [one] {$hours} hour
 *[other] {$hours} hours
  } in { $language } with a personal goal
-help-reach-hours-general-pluralized = Help Common Voice reach { NUMBER($hours) -> 
+help-reach-hours-general-pluralized = Help Common Voice reach { NUMBER($hours) ->
   [one] {$hours} hour
   *[other] {$hours} hours
  } in a language with a personal goal
 set-a-goal = Set a goal
 cant-decide = Can't decide?
-activity-needed-calculation-plural = { NUMBER($totalHours) -> 
-  [one] {$totalHours} hour
-  *[other] {$totalHours} hours
- } is achievable in just over { NUMBER($periodMonths) -> 
+activity-needed-calculation-plural-v2 = { NUMBER($totalHours) ->
```

Comment:
Since the old one string contains grammatical error, you can keep the string ID and replace with the correction in grammar.